### PR TITLE
add compiler pass for interface

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

File: UserBundle/Resources/config/resolve_document.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+resolve_target_documents:
+    Symfony\Component\Security\Core\User\UserInterface: OpenOrchestra\UserBundle\Document\User
```
Comment (nicolasThal) [inline UserBundle/Resources/config/resolve_document.yml]: can you also add the default configuration for the groupInterface ?

**Comment by itkg-alavieille** [comment]
@nicolasThal

**Comment by nicolasThal** [comment]
:+1:

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**